lang/lua-lsqlite3/Makefile

```diff
@@ -54,4 +73,10 @@ define Package/lsqlite3/install
 	$(CP) $(PKG_BUILD_DIR)/*.so $(1)/usr/lib/lua/
 endef
 
+define Package/lsqlite3-lua5.3/install
+	$(INSTALL_DIR) $(1)/usr/local/lib/lua/5.3
```

Comment:
`/usr/local`? this path seems unused on openwrt

Reply:
The default prefix is `/usr/local` in Lua5.3.

Reply:
![image](https://github.com/openwrt/packages/assets/28673295/af8974d2-042c-4791-92c9-0dde66db3e41)

Reply:
It might be better to patch the [lua5.3](https://github.com/openwrt/openwrt/tree/main/package/utils/lua5.3) package I think, just like what we did in [lua5.1](https://github.com/openwrt/openwrt/blob/main/package/utils/lua/patches/200-lua-path.patch).

Reply:
There are some packages already use it. Such as luasocket-lua-53.

Reply:
Hmmm okay.